style1.css

```diff
@@ -13,14 +13,20 @@
   position: absolute;
   width: 20px;
   height: 14px;
-  right: 15px;
   top: 19px;
 }
 
+.ic_menu > button {
+  background-color: white;
+  border: none;
+  margin-left: 280px;
+}
+
 .flex-container {
   display: flex;
   align-items: center;
   position: absolute;
+  width: 100%;
 }
 
```

Comment:
- [ ] GOOD JOB :clap: your design is looking great. However, you still need to work on the margin of the `project` in my work section. check the screenshots down here :point_down: 

you design     | the original design
------------------ | -----------------------
![Screenshot from 2021-11-26 22-13-10](https://user-images.githubusercontent.com/65068771/143626758-ecba75e5-f049-48fa-b93c-0ad7cef76572.png) | ![Screenshot from 2021-11-26 22-14-01](https://user-images.githubusercontent.com/65068771/143626805-024b9d6e-eacb-4864-a335-c09283dccec5.png)